test/expect/TestScript.test_milstm_fusion_cuda-backward.expect

```diff
@@ -91,16 +91,17 @@ with prim::FusionGroup_1 = graph(%0 : Float(*, *)
   %6 : Float(*, *) = aten::add(%3, %4, %5)
   return (%6);
 }
-with prim::FusionGroup_2 = graph(%0 : Float(*)
-      %1 : Float(*, *)
-      %2 : Float(*)
+with prim::FusionGroup_2 = graph(%0 : Float(*, *)
+      %1 : Float(*)
+      %2 : Float(*, *)
       %3 : Float(*, *)
-      %4 : Float(*, *)) {
-  %5 : Float(*, *) = aten::mul(%1, %4)
-  %6 : Float(*, *) = aten::mul(%5, %3)
-  %7 : Float(*, *) = aten::mul(%1, %2)
-  %8 : Float(*, *) = aten::mul(%5, %0)
+      %4 : Float(*)) {
+  %5 : Float(*, *) = aten::mul(%2, %4)
+  %6 : Float(*, *) = aten::mul(%2, %3)
+  %7 : Float(*, *) = aten::mul(%6, %1)
+  %8 : int = prim::Constant[value=1]()
```

Comment:
I am confused as to where this came from (there is probably some AD formula that is not working too well) but it is unrelated to this PR